config/control-plane/kustomization.yaml

```diff
@@ -110,6 +113,83 @@ transformers:
     kind: ConfigMap
     name: klm-dashboard-status
     version: v1
+- |-
```

Comment:
If we consider `control-plane` profile purely for KCP landscape, can we get rid of this `PatchTransformer`?

Reply:
which PatchTransformer are you referring to here?

Reply:
But the thing is we have the namespacetransformer, that's why we still need to patch the namespaces for these resources